versions.sh

```
@@ -1,6 +1,4 @@
-PG_VERSION=17.0
-PATRONI_VERSION=4.0.2
-CITUS_VERSION=12.1.4
-WAL_G_VERSION=3.0.3
+PG_VERSION=17.4
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)